src/main/kotlin/no/nav/medlemskap/regler/v1/Hovedregler.kt

```diff
@@ -1,12 +1,19 @@
 package no.nav.medlemskap.regler.v1
 
 import no.nav.medlemskap.domene.Datagrunnlag
-import no.nav.medlemskap.regler.common.*
+import no.nav.medlemskap.regler.common.Resultat
+import no.nav.medlemskap.regler.common.konklusjonIdentifikator
+import no.nav.medlemskap.regler.common.sjekkRegelsett
+import no.nav.medlemskap.regler.common.uavklartKonklusjon
 
 class Hovedregler(datagrunnlag: Datagrunnlag) {
 
     private val resultatliste: MutableList<Resultat> = mutableListOf()
-    private val reglerForRegistrerteOpplysninger = ReglerForRegistrerteOpplysninger(datagrunnlag)
+    private val reglerForRegistrerteOpplysninger = ReglerForRegistrerteOpplysninger(
```

Comment:
Ja, dette kan jeg like! Gjør gjerne dette for resten av kodebasen også, så blir det tydelig hva som er datagrunnlaget for hver enkelt regel, og det blir lettere å skrive tester uten å måtte finne på masse tulledata. Bra jobba!